public/components/application_analytics/components/flyout_components/service_detail_flyout.tsx

```diff
@@ -135,8 +132,7 @@ export function ServiceDetailFlyout(props: ServiceFlyoutProps) {
 
     setIsServicesDataLoading(true);
     Promise.all([
-      handleServiceViewRequest(serviceName, http, serviceDSL, setFields, mode),
-      handleServiceMapRequest(http, serviceDSL, mode, '', setServiceMap),
+      handleServiceViewRequest(serviceName, http, serviceDSL, setFields, mode, setServiceMap),
     ]).finally(() => setIsServicesDataLoading(false));
```

Comment:
it's one promise, you can remove the Promise.all

Reply:
Removed, thanks!